### DISCUSSION TITLE
Questions about Screentask

### DISCUSSION THREAD

**Discussion by shodanx2** [EslaMx7/ScreenTask#53] [General]
Hello,

I have a few questions regarding this application

What are the video codec used
Is hardware encoding and decoding usable ?
Is it possible to use dedicated viewer software such as VLC and ffplay for viewer desktop 
What is the achievable performance on gigabit ethernet ?
Expected latency at 1920x1080 ? Frame rate 30, 60 120 fps ?
Is multicast possible (for watching on many monitors over wifi but only one data stream instead of one per client !)

Can the app create new indirect displays and share them as new monitors ?

https://docs.microsoft.com/en-us/windows-hardware/drivers/display/indirect-display-driver-model-overview
https://github.com/microsoft/Windows-driver-samples/blob/master/video/IndirectDisplay/README.md

open source example driver
https://github.com/roshkins/IddSampleDriver

amyuni indirect display driver (freeware)
https://www.amyuni.com/forum/viewtopic.php?t=3030

**Comment by robssky** [comment]
Hi,

I used the previous version which asked me to upgrade to v1.2 which I did. Now the webpage has a header an footer and to display fullscreen I need to select the option on the bottom.

Is there a way to display fullscreen without pressing the "Fullscreen" button as the application I am wanting to use this for wont allow user interaction.

Thanks.

Regards
Rob.

**Reply by robssky** [reply]
Hi,

Just realized I can copy the old html file across from the previous version to the new version and that seems to work.

Thanks anyway.

Regards
Rob.
